modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -56,7 +61,92 @@ test.describe('Manage object fields through Model Builder', () => {
 		);
 
 		await apiHelpers.listTypeAdmin.deleteListTypeDefinition(
-			ListTypeDefinition.id
+			listTypeDefinition.id
+		);
+	});
+
+	test('all picklist definitions are listed during object field creation', async ({
+		apiHelpers,
+		modelBuilderPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		const listTypeDefinitions = await Promise.all(
+			Array(22)
+				.fill(null)
+				.map(() =>
+					apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
+				)
 		);
+
+		listTypeDefinitions.map((typeDefinition) =>
+			createdEntities.listTypeDefinitionIds.push(typeDefinition.id)
```

Comment:
Making so much pushes costs more. Try this:

```
createdEntities.listTypeDefinitionIds = listTypeDefinitions.map(({id}) => id);
```
.map() returns a new array that you assign in one step to createdEntities.listTypeDefinitionIds.